packages/web/public/static/locales/en/search.json

```diff
@@ -1,7 +1,7 @@
 {
   "filter": "Filter",
   "filterOnlyPublic": "Only public technologies",
-  "foundTerms": "{{nbHits}} result(s) found for the search term \"{{termQuery}}\" in {{timeSpentMS}}ms",
+  "foundTerms": "{{nbHits}} result(s) found in {{timeSpentMS}}ms",
```

Comment:
Had to remove search term from stats due to react-autosuggest. It changes search term while navigating through suggestions.